<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDatabaseDialog</class>
 <widget class="QDialog" name="EditDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHost">
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPort">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDatabase">
         <property name="text">
          <string>Database:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelUser">
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="editName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editHost"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPort">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editDatabase"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editUser"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editPassword"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
